commands/create_stack.go

```diff
@@ -33,10 +36,13 @@ func createStack() *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVar(&flags.config, "config", "", "path to a stack descriptor file (required)")
-	cmd.Flags().StringVar(&flags.buildOutput, "build-output", "", "path to output the build image OCI archive (required)")
-	cmd.Flags().StringVar(&flags.runOutput, "run-output", "", "path to output the run image OCI archive (required)")
+	cmd.Flags().StringVar(&flags.buildOutput, "build-output", "", "path or image reference to output the build image OCI archive (required)")
```

Comment:
Should this change be reverted - i.e. should this flag now only refer to the OCI archive location?

Reply:
Good catch. I'll change that back.